src/components/ens-registration/PendingRegistrations/PendingRegistrations.tsx

```diff
@@ -0,0 +1,166 @@
+import lang from 'i18n-js';
+import React, { useCallback } from 'react';
+import LinearGradient from 'react-native-linear-gradient';
+import { Alert } from '../../../components/alerts';
+import ButtonPressAnimation from '../../../components/animations/ButtonPressAnimation';
+import ImageAvatar from '../../../components/contacts/ImageAvatar';
+import {
+  AccentColorProvider,
+  Box,
+  Column,
+  Columns,
+  Divider,
+  Inset,
+  Stack,
+  Text,
+} from '@rainbow-me/design-system';
+import { RegistrationParameters } from '@rainbow-me/entities';
+import { REGISTRATION_MODES } from '@rainbow-me/helpers/ens';
+import {
+  useENSPendingRegistrations,
+  useENSRegistration,
+} from '@rainbow-me/hooks';
+import { useNavigation } from '@rainbow-me/navigation';
+import Routes from '@rainbow-me/routes';
+import { colors } from '@rainbow-me/styles';
+
+const PendingRegistration = ({
+  registration,
+  removeRegistration,
+  avatarUrl,
+}: {
+  avatarUrl?: string;
+  registration: RegistrationParameters;
+  removeRegistration: (name: string) => void;
+}) => {
+  const { navigate } = useNavigation();
+  const { startRegistration } = useENSRegistration();
+
+  const onFinish = useCallback(
+    async (name: string) => {
+      startRegistration(name, REGISTRATION_MODES.CREATE);
+      setTimeout(() => {
+        navigate(Routes.ENS_CONFIRM_REGISTER_SHEET, {});
+      }, 100);
+    },
+    [navigate, startRegistration]
+  );
+
+  const onRemove = useCallback(
+    async (name: string) => {
+      removeRegistration(name);
+    },
+    [removeRegistration]
+  );
+
+  return (
+    <Box>
+      <Columns alignVertical="center">
+        <Column width="content">
+          <Box paddingRight="10px">
+            <ImageAvatar image={avatarUrl || ''} size="small" />
+          </Box>
+        </Column>
+        <Column>
+          <Box>
+            <Text color="primary" numberOfLines={1} size="16px" weight="heavy">
+              {registration.name}
+            </Text>
+          </Box>
+        </Column>
+        <Column width="content">
+          <Box paddingRight="15px">
+            <ButtonPressAnimation
+              onPress={() => onFinish(registration.name)}
+              scaleTo={0.9}
+            >
+              <Box
+                alignItems="center"
+                as={LinearGradient}
+                borderRadius={16}
+                colors={colors.gradients.transparentToAppleBlue}
+                end={{ x: 0.6, y: 0 }}
+                height="30px"
+                justifyContent="center"
+                start={{ x: 0, y: 0.6 }}
+              >
+                <Inset horizontal="10px">
+                  <AccentColorProvider color={colors.appleBlue}>
+                    <Text color="accent" size="16px" weight="heavy">
```

Comment:
```suggestion
                    <Text color="action" size="16px" weight="heavy">
```

Think we can remove `AccentColorProvider` too and just have `color="action"`

Reply:
you're right!